docs/user_guide/sending/serialization.md

```diff
@@ -303,4 +303,6 @@ assert dv_from_proto_numpy.tensor_type == NdArray
 assert isinstance(dv_from_proto_numpy.tensor, NdArray)
 ```
 
+!!! note
+    Currently, union type hints are not supported for serialization to protobuf formats.
```

Comment:
Any union type? or Union types of regular types are? Can u add tests for the exceptions?

Reply:
I'll add more tests. It should be any union type as union type doesn't have the `from_protobuf` method